openupgradelib/openupgrade.py

```diff
@@ -26,11 +26,16 @@
 from openerp import release, tools, SUPERUSER_ID
 from openerp.tools.yaml_import import yaml_import
 from openerp.osv import orm
-from openerp.tools.mail import plaintext2html
 from openerp.modules.registry import RegistryManager
 import openerp.osv.fields
-import openerp.fields
 from . import openupgrade_tools
+if release >= "7.0":
+    from openerp.tools.mail import plaintext2html
+if release >= "8.0":
+    from openerp.fields.Many2many import Many2many, One2many
```

Comment:
Yeah, it is, but this string comparison is also good and without problems, so I prefer instead importing more variables (that are internals I think and can change)